### PR TITLE
 a simple expample

### PR DESCRIPTION
**Pull request by paribaker**
Just a quick example for folks to get started without any distractions

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -42,6 +44,163 @@ yarn add @thinknimble/tn-models-fp
...
+  client: axios.create(), // a client of your choice
+  baseUri: "api/users/", // a base URI to be used as a default 
+  models: {
+    /**
+     * since my create shape is different than the default AccountEntity 
+     * I can override it here
+     * 
+     * */ 
+    
+    create: createShape,
```
Comment (lakardion) [inline README.md:168]: This is actually the input shape for the creation of resources. It does not default to `entity` shape. 
If you don't pass it you won't see the `create` as a built-in method when inspecting the created api.

Not passing it means you will have to create your custom call if you want to do the create on your own

sample:
image

image

**Review by lakardion — APPROVED** [review]
Thanks this is great!

I need to put some time to improve the docs this gives me a great starting point

File: README.md (hunk 1 above)
Comment (paribaker) [reply README.md:168]: ah so the entity is the default output shape?

File: README.md [hunk 2]
```diff
@@ -42,6 +44,163 @@ yarn add @thinknimble/tn-models-fp
...
+    return fromApi(res.data)
+  }
+)
+/**
+ * There is no need for an output shape in this case
+ */
+const delete = createCustomServiceCall(
+  {
+    inputShape: {id:z.string().uuid()}
+  },
```
Comment (paribaker) [inline README.md:151]: typo here should be input

**Pull request merged**